management/pom.xml

```diff
@@ -148,6 +148,11 @@
             <artifactId>snmp4j</artifactId>
         </dependency>
         <dependency>
+            <groupId>uk.co.real-logic</groupId>
+            <artifactId>Agrona</artifactId>
+            <version>0.3.1</version>
```

Comment:
Move this to dependencyManagement section of top level pom.  And then remove the version